src/components/MaterialTheme/TwelveHoursMode.jsx

```diff
@@ -139,10 +177,13 @@ class TwelveHoursMode extends React.PureComponent {
 
     const HourPickerPointGenerator = pickerPointGenerator('hour', 12);
     const MinutePickerPointGenerator = pickerPointGenerator('minute', 12);
+    const localMessages = this.languageData();
```

Comment:
Updated this meridiem section to always return an internationalized version of the meridiem.